Cargo.toml

```diff
@@ -21,6 +21,7 @@ serde = { version = "1.0", optional = true, features = ["derive"] }
 [features]
 default = ["use_std"]
 use_std = []
+never_type = []
```

Comment:
If we should have an flag for an experimental feature in Rust, the flag should have the prefix `unstable-` to its feature name